app/models/loader/types.rb

```diff
@@ -170,16 +185,33 @@ def identifier
     class Group < Record
       def_delegators :@policy_object, :gidnumber
 
+      def verify; end
+
       def create!
         self.annotations ||= {}
         self.annotations["conjur/gidnumber"] ||= self.gidnumber if self.gidnumber
 
         super
       end
     end
-    
+
     class User < Record
-      def_delegators :@policy_object, :public_keys, :account, :role_kind, :uidnumber, :restricted_to
+      def_delegators :@policy_object, :public_keys, :account, :role_kind, :uidnumber, :uidnumberabc, :restricted_to
```

Comment:
what does `uidnumberabc` stand for?

Reply:
The Ruby convention would be something like `uid_number_abc`

Reply:
field removed